src/EnergyPlus/OutputProcessor.cc

```diff
@@ -4644,7 +4644,7 @@ namespace OutputProcessor {
             NumberOut = "0.0";
         } else {
             NumberOut = format("{:N}", repVal);
-            strip_trailing_zeros(trim(NumberOut));
+            strip_trailing_zeros(strip(NumberOut));
```

Comment:
Actual crash. v9.3.0 uses strip. Then during the move away from gio::write, it was first removed then added back as trim instead of strip

cf https://github.com/NREL/EnergyPlus/blob/baff08990c9c4d12d8bbde0378073018a3ea6acf/src/EnergyPlus/OutputProcessor.cc#L5290

https://github.com/NREL/EnergyPlus/commit/5dee4a244ec0e1af3c46b4e2d1efc3ee37a8eed4#diff-bd9d5281a7fe99bacbcd3d7112772f7804d8e091f10a56a0954a7639c24384fcR4946

Reply:
:+1: